docs/guide-tequila-api-key.md

```diff
@@ -1,6 +1,20 @@
 ---
-title: "Guide: Get your Tequila API key"
-sidebar_label: "Guide: Get your Tequila API key"
+title: Guide: Get your Tequila API key
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
```

Comment:
Margarita fetches data from GraphQL server. This server is just a proxy of officially supported Tequila REST API. The REST API requires an API key to be included in every call. GraphQL server loads the value from an environment variable named `API_KEY`.

Reply:
done